Header header   
bool action_source   #0 is navigation, 1 is RL
float32 speed     #-1 is undefined
float32 acc       #-100 is undefined
int16 direction   #0 left, 1 right, -1 undefined